process_report/processors/prepayment_processor.py

```diff
@@ -0,0 +1,217 @@
+import sys
+import logging
+from dataclasses import dataclass
+
+import pandas
+
+from process_report import util
+from process_report.invoices import invoice
+from process_report.processors import discount_processor
+
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
+
+
+@dataclass
+class PrepaymentProcessor(discount_processor.DiscountProcessor):
+    IS_DISCOUNT_BY_NERC = True
+    PREPAY_DEBITS_S3_FILEPATH = "Prepay/prepay_debits.csv"
+
+    @property
+    def PREPAY_DEBITS_S3_BACKUP_FILEPATH(self):
```

Comment:
nitpick: Generally, all capital letters is reserved for constants. This isn't a constant.